packages/client/src/views/SysAdmin/Config/Certificates/Certificates.tsx

```diff
@@ -355,260 +350,190 @@ class CertificatesConfigComponent extends React.Component<Props, State> {
+    const deathLastModified =
+      offlineResources.templates.certificates.death.lastModifiedDate
+    const CertificateSection = {
+      title: intl.formatMessage(messages.listTitle),
+      items: [
+        {
+          id: 'birth',
+          label: intl.formatMessage(messages.birthTemplate),
+          value: intl.formatMessage(messages.birthDefaultTempDesc),
+          actionsMenu: (
+            <>
+              {offlineResources.templates.certificates.birth && (
+                <ToggleMenu
+                  id={`template-birth-action-menu`}
+                  toggleButton={<VerticalThreeDots />}
+                  menuItems={this.getMenuItems(
+                    intl,
+                    Event.BIRTH,
+                    offlineResources.templates.certificates.birth.definition,
+                    offlineResources.templates.certificates.birth.fileName
+                  )}
+                />
+              )}
+              <HiddenInput
+                ref={this.birthCertificatefileUploader}
+                id={`birth_file_uploader_field_${offlineResources.templates.certificates.birth.id}`}
+                type="file"
+                accept={ALLOWED_IMAGE_TYPE_FOR_CERTIFICATE_TEMPLATE.join(',')}
+                onChange={this.handleSelectFile}
+              />
+            </>
+          )
+        },
+        {
+          id: 'death',
+          label: intl.formatMessage(messages.deathTemplate),
+          value: intl.formatMessage(messages.deathDefaultTempDesc),
+          actionsMenu: (
+            <>
+              {offlineResources.templates.certificates.death && (
+                <ToggleMenu
+                  id={`template-death-action-menu`}
+                  toggleButton={<VerticalThreeDots />}
+                  menuItems={this.getMenuItems(
+                    intl,
+                    Event.DEATH,
+                    offlineResources.templates.certificates.death.definition,
+                    offlineResources.templates.certificates.death.fileName
+                  )}
+                />
+              )}
+              <HiddenInput
+                ref={this.deathCertificatefileUploader}
+                id={`death_file_uploader_field_${offlineResources.templates.certificates.death.id}`}
+                type="file"
+                accept={ALLOWED_IMAGE_TYPE_FOR_CERTIFICATE_TEMPLATE.join(',')}
+                onChange={this.handleSelectFile}
+              />
+            </>
+          )
+        }
+      ]
+    }
+    CertificateSection.items.map((item) =>
+      mapValues(item, (val) => {
+        if (val === Event.BIRTH) {
+          item.value = intl.formatMessage(messages.eventUpdatedTempDesc, {
+            lastModified: birthLastModified
+          })
+        } else if (val === Event.DEATH) {
+          item.value = intl.formatMessage(messages.eventUpdatedTempDesc, {
+            lastModified: deathLastModified
+          })
+        }
+      })
```

Comment:
This bit gets me a bit confused. Is there a benefit in looping through the object keys instead of doing

```ts
if(item.id === Events.BIRTH) {
  item.value = intl.formatMessage(messages.eventUpdatedTempDesc, {
    lastModified: birthLastModified
  })
}
```

?

Also, if you're using a map, you should always return an updated value (and not mutate the value) as `.map` indicates to the code reader that there's a transformation from Array<A> to Array<B>

so this should look more like 

```ts
const itemsWithLastModified = CertificateSection.items.map((item) => {
  return {
    ...item,
    value: intl.formatMessage(messages.eventUpdatedTempDesc, {
      lastModified: birthLastModified
    })
  }
}
```